gulpfile.js

```diff
@@ -106,6 +106,7 @@ declareExtension('amp-viz-vega', '0.1', true);
 declareExtension('amp-google-vrview-image', '0.1', false, 'NO_TYPE_CHECK');
 declareExtension('amp-viewer-integration', '0.1', false);
 declareExtension('amp-youtube', '0.1', false);
+declareExtension('amp-hulu', '0.1', false, 'NO_TYPE_CHECK');
```

Comment:
nit, please put this in alphabetical order (even though some are not, lets try to keep it consistent).